Modules/core/BRPeerManager.c

```diff
@@ -2054,6 +2058,23 @@ const ChainParams *BRPeerManagerChainParams(BRPeerManager *manager) {
     return manager->params;
 }
 
+void PeerManagerGetAssetData(BRPeerManager *manager, void *infoManager, char *assetName, size_t nameLen,
+                             void (*receivedAssetData)(void *info, BRAsset *asset)) {
+    
+    for (size_t i = array_count(manager->connectedPeers); i > 0; i--) {
+        BRPeer *peer = manager->connectedPeers[i - 1];
+        
+        if (BRPeerConnectStatus(peer) != BRPeerStatusConnected) continue;
+        
+        peer->assetCallbackInfo = infoManager;
+        
```

Comment:
Can we put this constraint back in?

```suggestion
if (BRPeerVersion(peer) < 70018) continue;
```

Reply:
can we use this for debug
        if (BRPeerVersion(peer) >= 70018) {
            BRPeerSendGetAsset(peer, assetName, nameLen, receivedAssetData);
        } else
            peer_log(peer, "node doesn't support Assets MSG Protocol");

Reply:
Sure